README.md

```diff
@@ -131,29 +164,20 @@ Awesome DeFi apps you can deploy on Akash
 - [Speedtest Tracker](speedtest-tracker)
 
 ### Blockchain
+
 - [Bitcoin](bitcoin)
 - [Prysm Beacon](prysm-beacon)
 - [Substrate Node](substrate-node)
 - [Near Node](near)
 - [Vidulum](vidulum)
 - [Ethereum 2.0](Ethereum_2.0)
-- [POKT Network](pokt_network)
+- [POKT Network](pokt-network)
 - [Polkadot](polkadot)
 - [Kadena](Kadena)
 - [Bitcoin Cash Node](bitcoincashnode)
 - [Handshake](handshake)
 - [Fuse Network Node](fuse-network-node)
 
-### Blogging
```

Comment:
please keep the blogging section

Reply:
It is a copy-paste of line 80

![Screenshot 2023-08-02 at 1 41 02 PM](https://github.com/akash-network/awesome-akash/assets/1892194/2bf0982e-91a7-4c97-9996-8b753978b600)

Reply:
@arno01 this appears to be a duplicate; hence it was deleted.

You can actually see “Blogging” appear twice on the `master` branch currently.

Is there anything else blocking this PR?

Thanks!